README.md

```diff
@@ -1,302 +1,293 @@
-# Origami Tracking component
+# o-tracking
 
-Include in your product to send tracking requests to the Spoor API.
+Include in your product to send tracking requests to the [Spoor API](https://spoor-docs.herokuapp.com/).
 
-- [The spoor ecosystem](#the-spoor-ecosystem)
 - [Usage](#usage)
-- [Events](#events)
-- [Parameters](#parameters)
+- [How To View The Data](#how-to-view-the-data)
+- [Example Implementations](#example-implementations)
 - [Migration Guide](#migration-guide)
 - [Contact](#contact)
 - [Licence](#licence)
 
-## The Spoor ecosystem
-![ScreenShot](https://rawgit.com/Financial-Times/o-tracking/master/resources/images/ngda-system-design.svg)
+## Usage
 
-[Spoor API docs](https://spoor-docs.herokuapp.com/)
+Check out [how to include Origami components in your project](https://origami.ft.com/docs/components/#including-origami-components-in-your-project) to get started with `o-tracking`.
 
-## Usage
+### How to view the data
+
+Once you have sent data into Spoor, you can view the data via Chartio.
+The FT has a Slack channel named #chartio_q_a where anyone can get help with Chartio.
+If you have never used Chartio before, you will need to [request access via this form](https://docs.google.com/forms/d/e/1FAIpQLSc1z1hTtnhe4KSXuZ220Zf53wTm0ucYVf2B3jYY5ZSyNW2VSw/viewform).
+To sign in to Chartio:
+- Open the [FT Chartio](https://chartio.com/financialtimes/) in your browser.
+- Press the "Sign in with Google" button and select your FT Google Account.
+
+### How to track extra click data using `data-trackable` attributes
```

Comment:
I think this section might make more sense after basic setup instructions

Reply:
Fixed in #309